data/pages-content/jobs-board.md

```diff
@@ -3,32 +3,340 @@ title: Jobs Board
+- ...and you don't mind collecting bonus points with your cloud and Kubernetes skills
 
+<h5>Technologies you'll use</h5>
+
+- Python 3.9/3.10/3.11
+- SimPy
+- Many Python libraries across various domains
+- PostgreSQL, MongoDB, Snowflake
+- RabbitMQ
+- AWS, Docker, Kubernetes, TeamCity
+- Microservices
+
+</details>
+
+[Apply here](https://picnic.app/careers/jobs/5005708/technology--amp--engineering/amsterdam-north-holland-netherlands/senior-warehouse-simulation-software-engineer)
+
+
+## [The Mobility House GmbH](https://www.mobilityhouse.com/)
+We are a leading provider of smart charging solutions with many years of experience in managing battery storage in energy and power markets, we will work with you to unlock the potential that lies in innovative electric vehicle energy products such as V2G (vehicle to grid).
+
+<details>
+  <summary><h4 id="mobility-house1-teamlead-technology-vgi">Teamlead Technology - VGI</h4></summary>
+  - You lead a team of talented SW engineers. Your team's focus is on developing full-stack solutions to control and interlink Electric Vehicle Charging Processes. 
+  - You design, plan and implement information and infrastructure systems to support business strategies by interacting with assets as well as users.
+  - You are supervising, mentoring & scaling the team, putting emphasis on security, stability and scalability as well as on quality measures such as TDD.
+  - You interact with peers and cross-functional colleagues to fullfil our joint mission towards #NorthStar
 </details>
 
-[Apply here](http://example.com) or contact John Doe: example@example.com
+[Apply here](https://www.mobilityhouse.com/de_de/karriere) or contact Maren Lueess: maren.lueess@moblityhouse.com
 
 <details>
-  <summary><h3 id="company-job3">Company Job3</h3></summary>
+  <summary><h4 id="mobility-house2-senior-software-engineer">(Senior) Software Engineer</h4></summary>
+  - You design, develop, test, deploy, maintain and improve software for our energy storage solutions that enable the integration of electric vehicles into the power grid
+  - You contribute to the continuous improvement of our software development technologies, tools and processes to ensure highest quality in our products
+
+**Our teams**:
+- Development and CI/CD of a smart charging solution for managing load distribution on EV fleets
+- Development of a smart charging schedule and EV insights solution for individual electric vehicles to reduce CO2 footprint and charging costs
+- Development of software solutions to provide grid services by utilizing aggregated EV batteries in stationary storages
+
+</details>
 
-  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+[Apply here](https://www.mobilityhouse.com/de_de/karriere) or contact Maren Lueess: maren.lueess@moblityhouse.com
```

Comment:
The same here, I think we can fix the link to: https://www.mobilityhouse.com/int_en/career/senior-software-engineer-m-w-d-341.html

Reply:
as as above! Fixed with thanks!